### DISCUSSION TITLE
SWS 9898

### DISCUSSION THREAD

**Discussion by DLHN-cz** [schizza/SWS-12500-custom-component#11] [Q&A] [answered]
Hi, I woulkd like to ask, if this component supports only SWS 12500, or if it's usable also for the cheaper version, SWS 9898?

**Comment by schizza** [comment]
Hi, it depends on how you can setup the station in AP mode. If there is an option for sending data to server of your choice, then it will work as well as in SWS 12500.
But on the manufacturer's page in Manual, it seems, that the station does not support this feature. 
Post me screenshot of your current AP and firmware.

**Reply by DLHN-cz** [reply]
Unfortunatelly I haven't it. Just looking for it and thinking, if wouldn't be better to take the cheaper one due to to same sensor HW, when I need to read values in HA only.

**Reply by schizza** [reply, marked as answer]
It seems, that 9898 does not support sending data to custom address, only to WeatherCloud and WUnderground.
![CleanShot 2024-03-27 at 09 58 23@2x](https://github.com/schizza/SWS-12500-custom-component/assets/4604900/3aa0344b-539e-4764-a4de-13608b3e9adc)

**Reply by Celer21** [reply]
Maybe it would be enough to only redirect DNS weathercloud/wunderground to IP of Home assistant instance?

**Reply by schizza** [reply]
Redirecting DNS record to HA will work only if your HA instance is listening on port 80. And I am afraid, that home assistant will be listening for domain name configured in settings. So if you will pass wundegroud.com it will not be accepted. 
The trick might be using proxy, but this is the way I wanted to avoid.
